tests/test_production_scenarios.py

```
@@ -0,0 +1,412 @@
+    def test_system_health_check_scenario(self):
+        """システムヘルスチェックのシナリオ"""
+        # Scenario: GitHub Actionsで定期的に実行されるヘルスチェック
+
+        # Act - 各コンポーネントの状態を確認
+        health_status = {"storage": False, "config": False, "fetcher": False}
+
+        # ストレージチェック
+        try:
+            self.storage._ensure_directories()
+            health_status["storage"] = True
+        except:
```

Ruff (0.14.2)

Do not use bare `except`

(E722)